test/Erc20Quest.spec.ts

```diff
@@ -334,33 +334,39 @@ describe('Erc20Quest', async () => {
       expect(await deployedSampleErc20Contract.balanceOf(owner.address)).to.equal(totalRewardsPlusFee * 100)
     })
 
-    it('should transfer non-claimable rewards back to owner', async () => {
-      await deployedFactoryContract.connect(firstAddress).mintReceipt(questId, messageHash, signature)
+    it('should transfer correct amount non-claimable rewards to address', async () => {
+      await deployedRabbitholeReceiptContract.setMinterAddress(deployedFactoryContract.address)
+
+      await deployedFactoryContract.mintReceipt(questId, messageHash, signature)
       await deployedQuestContract.start()
       await ethers.provider.send('evm_increaseTime', [expiryDate + 100])
-      await deployedQuestContract.connect(firstAddress).claim()
-
+      await deployedQuestContract.connect(owner).claim()
```

Comment:
We should try to move the `claim()` to after `withdrawRemainingTokens()` cause we want to ensure that even after withdraws that users can still claim rewards if they haven't

Reply:
Good idea, I modified the test to do that.